testing/acceptance_test.py

```diff
@@ -626,13 +626,14 @@ def test_cmdline_python_namespace_package(self, testdir, monkeypatch):
         #             ├── __init__.py
         #             └── test_world.py
 
-        def join_pythonpath(*dirs):
+        def append_pythonpath(*dirs):
```

Comment:
This is actually prepending `dirs` to the `PYTHONPATH` right?

Reply:
True (also above).  Will rename it.